apps/src/aiEvaluation/aiEvaluationApi.ts

```diff
@@ -35,7 +39,28 @@ export interface StudentWorkEvaluation extends StudentAnswer, AIResponse {
   id: number;
 }
 
-export async function evaluateStudentWork(
+export async function evaluateFreeResponse(
+  studentAnswer: StudentAnswer,
+  levelId: number,
+  unitId: number
+): Promise<AIResponse> {
+  return evaluateStudentWorkOverall(studentAnswer, levelId, unitId);
+}
+
+export async function evaluateStudentCode(
```

Comment:
Optional: In general I prefer to expose the two functions (`evaluateStudentWorkOverall` and `evaluateStudentWorkSkills`) instead of having a wrapper with a boolean flag. I think it is a bit cleaner and easier to parse. But there can be good reasons we need the wrapper so I'll leave up to you.